openproblems/tasks/cell_cell_communication/datasets/allen_brain_atlas.py

```diff
@@ -0,0 +1,13 @@
+from ....data.allen_brain_atlas import load_mouse_brain_atlas
+from ....tools.decorators import dataset
+
+
+@dataset(
+    "~15k Murine Brain cells" "Tasic, B. et al., 2016. Nat. Neurosci. 19, 335–346 .",
+    data_url=load_mouse_brain_atlas.metadata["data_url"],
+    data_reference=load_mouse_brain_atlas.metadata["data_reference"],
+    dataset_summary="TODO",
```

Comment:
Please fill in a short (<280char) description

Reply:
Done.